### PR TITLE
Bug/5005 461 hero banner divider color

### LINKED ISSUE #461: Hero Banner divider color
#### Describe the bug / expected behavior
Hero banner divider color feels too light. This might just be a rendering/clipping issue — verify that it's using the correct color from designs.

From https://github.com/etn-ccis/blui-react-native-themes/issues/57

### PR DESCRIPTION
**Pull request by surajeaton**
Fixes #461 BLUI-5005

#### Changes proposed in this Pull Request:
- Updated hero banner divider color to be neutralVariant[50]

#### Screenshots / Screen Recording (if applicable)
- 
Simulator Screenshot - iphone 12 pro - 2023-12-12 at 15 24 06

#### To Test:
- yarn start

#### Any specific feedback you are looking for?
-

### CODE REVIEW

**Review by huayunh — COMMENTED** [review]
I don't think you will need a changelog item here. This bug was introduced as part of Material Design 3 overhaul in the unreleased RN component library v8.

File: components/src/core/HeroBanner/HeroBanner.tsx [hunk 1]
```diff
@@ -47,7 +48,7 @@ export const HeroBanner: React.FC<HeroBannerProps> = (props) => {
                 {childrenArray}
                 {divider && (
                     <View style={defaultStyles.divider}>
-                        <Divider />
+                        <Divider style={{ backgroundColor: BLUIColors.neutralVariant[50] }} />
```
Comment (huayunh) [inline components/src/core/HeroBanner/HeroBanner.tsx:51]: Use the tokens. I believe it should be outline variant

**Comment by ektaghag-eaton** [comment]
This will be fixed once we publish the RN themes package. Here is the fix that is not available in the latest alpha package https://github.com/etn-ccis/blui-react-native-themes/pull/61/files#diff-82eb8ee12983a50811096254a18527e233f6a506303e2dc7f35fc0307ac664d5R43

**Comment by surajeaton** [comment]
Tried replacing the node_modules in react showcase with the latest dev dist of react native themes:
Simulator Screenshot - iphone 12 pro - 2023-12-13 at 14 07 58

Closing this PR, as this will get fixed once react native themes package gets published.

**Pull request closed without merging**